com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/RenderPipelineSettings.cs

```diff
@@ -101,7 +101,8 @@ internal static RenderPipelineSettings NewDefault() => new RenderPipelineSetting
             decalLayerName7 = "Decal Layer 7",
             msaaSampleCount = MSAASamples.None,
             supportMotionVectors = true,
-            supportRuntimeDebugDisplay = true,
+            supportRuntimeDebugDisplay = false,
```

Comment:
Is it intended to change the default here for debug display?

Reply:
yes. We shouldn't compile runtimedebugdisplay by default, big overred for compil time